easybuild/easyconfigs/l/lz4/lz4-1.8.2-GCCcore-5.4.0.eb

```diff
@@ -0,0 +1,31 @@
+easyblock = 'ConfigureMake'
+
+name = 'lz4'
+version = '1.8.2'
+
+homepage = 'https://lz4.github.io/lz4/'
+description = """LZ4 is lossless compression algorithm, providing compression speed at 400 MB/s per core. 
+ It features an extremely fast decoder, with speed in multiple GB/s per core."""
+
+toolchain = {'name': 'GCCcore', 'version': '5.4.0'}
+
+source_urls = ['https://github.com/lz4/lz4/archive/']
+sources = ['v%(version)s.tar.gz']
+checksums = ['0963fbe9ee90acd1d15e9f09e826eaaf8ea0312e854803caf2db0a6dd40f4464']
+
+builddependencies = [('binutils', '2.26')]
+
+skipsteps = ['configure']
+
+maxparallel = 4
```

Comment:
@vanzod Any specific reason for this?

Reply:
Yes, there are few issues filed for high parallel make. It looks like they managed to get it up and running correctly up to `6` threads, so I set `4` for safety.

Reply:
Is it worth including a reference to those known issues as a comment above this line?

Reply:
Does the value of `parallel` affects also the `make test`?